contracts/Auctioneer.sol

```diff
@@ -127,6 +134,7 @@ contract Auctioneer is CloneFactory {
         );
 
         openAuctions[cloneAddress] = true;
+        openAuctionsCount++;
```

Comment:
Until we are not on Solidity 0.8, I think it is safer to use SafeMath `.add(1)`, `sub(1)` for increment/decrement, just in case there is a code path when the operation can overflow.

Side note, we need to standardize Solidity version in our files, I've just opened an issue for that, #98.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/e010a6fea5ef1f0aad862a924727c29b87fb2bf3